gradoop-common/src/test/java/org/gradoop/common/model/impl/pojo/GraphElementTest.java

```diff
@@ -0,0 +1,39 @@
+package org.gradoop.common.model.impl.pojo;
+
+import org.gradoop.common.model.impl.id.GradoopId;
+import org.gradoop.common.model.impl.id.GradoopIdSet;
+import org.gradoop.common.model.impl.properties.Properties;
+import org.junit.Test;
+import org.mockito.Mockito;
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.CALLS_REAL_METHODS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.withSettings;
+
+public class GraphElementTest {
+
+  @Test
+  public void testAddGraphIdNoGraphIds() {
+    GraphElement graphElementMock = mock(GraphElement.class, Mockito.CALLS_REAL_METHODS);
+
+    GradoopId id = new GradoopId();
```

Comment:
This constructor is only used for serialization, please use `GradoopId.get()` instead.

Reply:
done